subprojects/platform-jvm/src/main/java/org/gradle/platform/Architecture.java

```diff
@@ -25,5 +25,5 @@
  */
 @Incubating
 public enum Architecture {
-    I386, AMD64, AARCH64
+    X32, X64, ARM32, ARM64
```

Comment:
As discussed, let's go with `X86`, `X86_64` and `AARCH64` which are closer to the values probed on recent JVM `os.arch` system property

Reply:
Already done